defaults/main.yml

```diff
@@ -83,10 +83,10 @@ owncloud_share_api_allow_public_notification: False
 # owncloud_share_api_public_notification_language: en # defaults to not set
 owncloud_share_api_allow_social_share: True
 owncloud_share_api_auto_accept_share: False
-owncloud_share_api_allow_resharing: False
+owncloud_share_api_allow_resharing: True
```

Comment:
https://github.com/owncloud/core/blob/master/lib/private/Share/Share.php#L1753

The default value in ownCloud core seems to be "yes" or "True" which means, setting this value to True is redundant. I would rather go and unset the value (comment it in this line and mention the default value in core)

In OCIS this behaviour is not yet implemented, so we could go there later and see what the default will be see here:
https://github.com/owncloud/ocis/issues/1328

In our customer deployments it seems to be 50/50 if this value is set to "yes" or "no" in the final configs, so I would not mind if we comment out the value here, so that it will be "yes" by default, but we should make it a BREAKING CHANGE not just a "chore:" PR to our ansible-repo as this might change behaviour in deployments massively!

Reply:
changed it to BREAKING CHANGE